pom.xml

```diff
@@ -277,6 +282,36 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
```

Comment:
You're right that the artifact does not exist in Hadoop 1. The API for the the mini cluster is different in that release too, so it would be tricky to support both. This is for testing, so it may be possible to exclude the test somehow, although it probably need a special Maven profile.